src/controllers/users.js

```diff
@@ -0,0 +1,34 @@
+import models from '../models';
+
+/**
+ * User controller Class
+ */
+export default class UserController {
+/**
+ * @name CreateUser
+ * @description Allows an admin add a new user
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {object} The API response
+ */
+  static async createUser(req, res) {
+    try {
```

Comment:
I would like for you to check if the user exists first before proceeding with creating the user